inginious/frontend/pages/course_admin/submissions.py

```diff
@@ -68,6 +70,8 @@ def POST_AUTH(self, courseid):  # pylint: disable=arguments-differ
 
                 tasks = course.get_tasks()
                 for submission in data:
+                    self.database.submissions.update_one(
+                        {"_id": submission['_id']}, {'$set': {"last_replay": datetime.now()}})
```

Comment:
I actually that the best way to put this piece of code is in the submission manager code.